zhaquirks/inovelli/VZM36.py

```diff
@@ -1,140 +1,18 @@
-                    Ota.cluster_id,
-                ],
-            },
-            2: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMABLE_LIGHT,
-                INPUT_CLUSTERS: [
-                    Basic.cluster_id,
-                    Identify.cluster_id,
-                    Groups.cluster_id,
-                    Scenes.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    Fan.cluster_id,
-                    Diagnostic.cluster_id,
-                    LightLink.cluster_id,
-                    INOVELLI_VZM36_CLUSTER_ID,
-                ],
-                OUTPUT_CLUSTERS: [
-                    Ota.cluster_id,
-                ],
-            },
-            242: {
-                PROFILE_ID: zgp.PROFILE_ID,
-                DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
-                INPUT_CLUSTERS: [],
-                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
-            },
-        },
-    }
-
-    replacement = {
-        ENDPOINTS: {
-            1: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMABLE_LIGHT,
-                INPUT_CLUSTERS: [
-                    Basic.cluster_id,
-                    Identify.cluster_id,
-                    Groups.cluster_id,
-                    Scenes.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    Diagnostic.cluster_id,
-                    LightLink.cluster_id,
-                    InovelliVZM36LightCluster,
-                    WWAH_CLUSTER_ID,
-                ],
-                OUTPUT_CLUSTERS: [
-                    Ota.cluster_id,
-                ],
-            },
-            2: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMABLE_LIGHT,
-                INPUT_CLUSTERS: [
-                    Basic.cluster_id,
-                    Identify.cluster_id,
-                    Groups.cluster_id,
-                    Scenes.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    Fan.cluster_id,
-                    Diagnostic.cluster_id,
-                    LightLink.cluster_id,
-                    InovelliVZM36FanCluster,
-                ],
-                OUTPUT_CLUSTERS: [
-                    Ota.cluster_id,
-                ],
-            },
-            242: {
-                PROFILE_ID: zgp.PROFILE_ID,
-                DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
-                INPUT_CLUSTERS: [],
-                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
-            },
-        },
-    }
-
-    device_automation_triggers = INOVELLI_AUTOMATION_TRIGGERS
+(
+    QuirkBuilder("Inovelli", "VZM36")
+    .replaces(InovelliVZM36LightCluster)
+    .replaces(InovelliVZM36FanCluster, endpoint_id=2, cluster_type=ClusterType.Client)
```

Comment:
`InovelliVZM36FanCluster` was in `INPUT_CLUSTERS` before, so the cluster type should be the default of `Server` (for input), right? I.e. this:

```suggestion
    .replaces(InovelliVZM36FanCluster, endpoint_id=2)
```

Reply:
Committed as suggested, yeah this would be an input cluster, I had them backwards I think.